server/utils/configs.js

```diff
@@ -127,9 +223,19 @@ async function testConfigOptions(options) {
     throw e;
   }
 
-  const clients = clientFactory(clientConfig);
+  const agents = new Map([
+    ['bcoin', 'bitcoin'],
+    ['bcash', 'bitcoincash'],
+    ['hsd', 'handshake']
+  ]);
 
   const clientErrors = new ClientErrors();
+  const chain = clientConfig.str('chain', 'bitcoin');
+
+  if (!pkg.chains.includes(chain))
```

Comment:
We now have 2 places where this functionality lives - one in bpanel-utils and one here

Reply:
Yeah, I think having the actual list in bpanel makes the most sense. bpanel-utils though can make a function available though to confirm compatibility (and that can just take in the current pkg file to check)